### PR TITLE
Hotfix 1.5.1

### PR DESCRIPTION
**Pull request by KochTobi**
## 1.5.1
* Bump `org.apache.logging:log4j:log4j-api` `2.13.2` -> `2.15.0`
* Bump `org.apache.logging:log4j:log4j-core` `2.13.2` -> `2.15.0`
* Fixes CVE-2021-4422

### CODE REVIEW

File: pom.xml [hunk 1]
```diff
@@ -57,7 +60,7 @@
 		<dependency>
 			<groupId>com.google.truth</groupId>
 			<artifactId>truth</artifactId>
-			<version>1.0.1</version>
+			<version>1.1.3</version>
```
Comment (jenniferboedker) [inline pom.xml:63]: why was this changed?

**Comment by jenniferboedker** [comment]
You might want to add some github actions when deleting the travis files. I removed the check from the branch since it does not run anyhow.

File: pom.xml (hunk 1 above)
Comment (KochTobi) [reply pom.xml:63]: because it can be increased and it should be updated.

**Review by Steffengreiner — APPROVED** [review]
LGTM 🚀

**Pull request merged**